src/nvim/popupmnu.c

```diff
@@ -436,7 +436,7 @@ void pum_redraw(void)
     const char *anchor = pum_above ? "SW" : "NW";
     int row_off = pum_above ? pum_height : 0;
```

Comment:
I would rather change the sign here so that `row_off` and `col_off` is consistent.

Reply:
It would be consistent on one level, and confusing on another.

We're really not `substracting pum_height` (which would be negative), we're adding pum_height.

If you think i'ts better, i'll change it.

Reply:
you could change `col_off` as well but having row and col be different is just weird.